.github/workflows/pipelines.yml

```diff
@@ -0,0 +1,89 @@
+---
```

Comment:
Rename this file to `.github/workflows/pipeline.yml` to match the other projects in this org.